HowToContribute.md

```diff
@@ -11,31 +11,31 @@ To build the project, run maven commands.
 ```bash
 git clone https://github.com/Microsoft/azure-spring-boot.git 
 cd azure-spring-boot
-mvnw clean install
+mvn clean install
 ```
 
 ## Test
 
 - Run unit tests
 ```bash
-mvnw clean install
+mvn clean install
 ```
 
-- Skip tests execution
+- Skip test execution
 ```bash
-mvnw clean install -DskipTests
+mvn clean install -DskipTests
```

Comment:
I think mvnw is still valid, any reasons to change it back to mvn?

Reply:
On windows, `mvnw` is fine. On mac, it has to be `./mvnw`, for `mvnw` is not recognizable. I don't want to complicate these commands by differentiating platforms.